package/boot/uboot-sunxi/patches/211-no-kwbimage.patch

```diff
@@ -0,0 +1,10 @@
+--- a/tools/Makefile
++++ b/tools/Makefile
+@@ -113,7 +113,6 @@ dumpimage-mkimage-objs := aisimage.o \
+ 			imximage.o \
+ 			imx8image.o \
+ 			imx8mimage.o \
+-			kwbimage.o \
```

Comment:
Please add a comment to the patch describing why you do not compile `kwbimage.o`.